### ISSUE TITLE
Note that newtype structs serialize and deserialize to the inner value with no wrapper

### ISSUE THREAD

**Issue by Mark-Simulacrum** [serde-rs/serde-rs.github.io#15]
This may already be recorded, but from a quick read through, I was unable to find it.

**Comment by dtolnay** [comment]
I added some JSON-specific information in https://serde.rs/json.html. I also added a note about newtype structs under https://serde.rs/impl-serialize.html#serializing-a-struct. I intend to write more detail about this as part of #3.

Let me know if there is a different place you would have expected to find information about tuple structs. I would like to treat JSON as just an example of a data format, because like most format-specific things this does not apply to every format. For example Bincode follows the zero-fluff approach of no wrappers anywhere, so it is misleading to say that `S { a: 0 }` is serialized with a wrapper but `S(0)` is serialized with no wrapper because they both serialize as just `0` in Bincode.
